specs/windows/bitlocker_info.table

```diff
@@ -7,5 +7,6 @@ schema([
   Column("persistent_volume_id", TEXT, "Persistent ID of the drive."),
   Column("conversion_status", INTEGER, "The bitlocker conversion status of the drive."),
   Column("protection_status", INTEGER, "The bitlocker protection status of the drive."),
+  Column("encryption_method", INTEGER, "The encryption type of the device."),
```

Comment:
Could you add some documentation of what the integer values will mean for the encryption method?

Reply:
Sure - should I list in the default method(s) for bitlocker or all 7 possible values?

Reply:
All 7 possible values I think is best. That way when people look up the documentation for this table it's easier for them to make sense of the results.

Reply:
I think this is the last thing we need and this PR should be G2G

Reply:
Just making sure we're 100% positive on this. Adding all 7 possible values and their corresponding numbers is going to make the result of .table bitlocker_info pretty huge. That's ok? For other fields I noticed that even if the WMI property has multiple numerical values (1,2,3,etc) it's not in the description of those tables.

Or what if I turned it into a switch that returns text equivalents to the encryption method values?

Reply:
I meant just in the documentation please enumerate the possible values and their meanings :p

A switch statement with text works equally well (if not better)!

Reply:
I think I'll do the switch. That will save people from having to look it up every time. Friendlier UX!

Reply:
Alrighty, implemented a switch to return the method as a string instead of just an integer